Server/src/services/dataProcess/IDataProcessService.ts

```diff
@@ -0,0 +1,22 @@
+export abstract class FileHandlerFactory {
+
+    abstract getFileHandler(filePath?: string): AbstractFileHandler
+}
+
+export abstract class AbstractFileHandler {
```

Comment:
file name doesnt correspond to file contents

Reply:
Will update